cli/cmd/entitlements_getcustomerrelease.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated/client/entitlements"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var entitlementsGetReleaseCustomerID string
+var entitlementsGetReleaseInstallationID string
+var entitlementsGetReleaseAPIServer string
+
+var entitlementsGetCustomerRelease = &cobra.Command{
+	Use:   "get-customer-release",
+	Short: "Preview the end customer release view",
+	Long: `Preview the end customer release view for a given customer ID + installation ID`,
+}
+
+func init() {
+	entitlementsCmd.AddCommand(entitlementsGetCustomerRelease)
+
+	entitlementsGetCustomerRelease.Flags().StringVar(&entitlementsGetReleaseCustomerID,"customer-id", "", "customer id")
+	entitlementsGetCustomerRelease.Flags().StringVar(&entitlementsGetReleaseInstallationID,"installation-id", "", "installation id")
+	entitlementsGetCustomerRelease.Flags().StringVar(&entitlementsGetReleaseAPIServer,"replicated-api-server", "https://pg.replicated.com/graphql", "Upstream api server")
```

Comment:
I think this one is a duplicate?

Reply:
almost -- this changes the previously-defined flag to default to `pg.` intead of `g.`. This command is a special case which simulates the customer view.

Reply:
Gotcha, hadn't noticed that
Might be a good comment to add

Reply:
```suggestion
	// change the default from g.replicated.com to pg.replicated.com
	entitlementsGetCustomerRelease.Flags().StringVar(&entitlementsGetReleaseAPIServer,"replicated-api-server", "https://pg.replicated.com/graphql", "Upstream api server")
```